### PR TITLE
added chat msg to reveal killer of locked team

### PR DESCRIPTION
**Pull request by timgame**
Also removed a "bug", the players killed each other recursively. This wasn't a problem before I added the messages though.

### CODE REVIEW

**Comment by east** [comment]
> Also removed a "bug", the players killed each other recursively.

What changed except that you added the chatmessages?

**Comment by east** [comment]
Okay, I got it now.
Looks good to me.

File: src/game/server/teams.cpp [hunk 1]
```diff
@@ -623,11 +623,21 @@ void CGameTeams::OnCharacterDeath(int ClientID, int Weapon)
 		SetForceCharacterTeam(ClientID, Team);
 
 		if (GetTeamState(Team) != TEAMSTATE_OPEN)
+		{
+			ChangeTeamState(Team, CGameTeams::TEAMSTATE_OPEN);
+
+			char aBuf[512];
+			str_format(aBuf, sizeof(aBuf), "Everyone in your locked team was killed because you %s.", Weapon == WEAPON_SELF ? "killed" : "died");
```
Comment (heinrich5991) [inline src/game/server/teams.cpp]: I think this message is a little long, just You died or 'Foobar' died should be fine. Also, I think the verb "kill" needs an object.

Comment (nzyuzin) [inline src/game/server/teams.cpp]: Maybe something like "X killed the team" for both cases?

**Comment by Soreu** [comment]
"Team died because of X" ?

File: src/game/server/teams.cpp (hunk 1 above)
Comment (Soreu) [inline src/game/server/teams.cpp]: "Team died because of X" sounds better imo

**Pull request merged**